### PR TITLE
Add developing on modular readme

### PR DESCRIPTION
**Pull request by bonicim**

### CODE REVIEW

File: DEVELOPING_ON_MODULAR.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## Existing Agent
+* Fork the Agent repo on Github
+* Clone the repo in your local environment
+* Setup the environment using poetry
+
+```shell
+poetry install 
+```
+
+* Create a working branch off of 'develop'
```
Comment (craig8) [inline DEVELOPING_ON_MODULAR.md:30]: Why are these three bulleted.

File: DEVELOPING_ON_MODULAR.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+* Make a code change
+* Run tests
+
+```shell
+poetry run pytest tests/
```
Comment (craig8) [inline DEVELOPING_ON_MODULAR.md:35]: shouldn't need to add tests folder

**Review by craig8 — CHANGES_REQUESTED** [review]
Looks good just a few questions.

File: DEVELOPING_ON_MODULAR.md (hunk 2 above)
Comment (bonicim) [reply DEVELOPING_ON_MODULAR.md:35]: This command doesn't add a tests folder. It runs the tests in the project using `pytest`.

File: DEVELOPING_ON_MODULAR.md (hunk 1 above)
Comment (bonicim) [reply DEVELOPING_ON_MODULAR.md:30]: They are bulleted to make it clear that to the user on how to start developing on an existing Agent; it tries to mirror the TDD process of working on legacy code.

File: DEVELOPING_ON_MODULAR.md (hunk 2 above)
Comment (craig8) [reply DEVELOPING_ON_MODULAR.md:35]: I am saying there is no need to specify tests

pytest will find them

Comment (craig8) [reply DEVELOPING_ON_MODULAR.md:35]: poetry run pytest tests/ is the same as poetry run pytest

File: DEVELOPING_ON_MODULAR.md (hunk 1 above)
Comment (craig8) [reply DEVELOPING_ON_MODULAR.md:30]: Ahh perhaps a header to say "the recommended development path is.... or something.  The bullets seem to spring up almost out of nowhere to me.

**Review by craig8 — CHANGES_REQUESTED** [review]

File: DEVELOPING_ON_MODULAR.md (hunk 2 above)
Comment (bonicim) [reply DEVELOPING_ON_MODULAR.md:35]: I see. I added the `tests/` parameter to make it for explicit, especially to readers who aren't so familiar with pytest.

File: DEVELOPING_ON_MODULAR.md (hunk 1 above)
Comment (bonicim) [reply DEVELOPING_ON_MODULAR.md:33]: Changed.

File: DEVELOPING_ON_MODULAR.md (hunk 2 above)
Comment (craig8) [reply DEVELOPING_ON_MODULAR.md:38]: that works

**Comment by bonicim** [comment]
Amended last commit to reflect correct author.

**Review by craig8 — APPROVED** [review]

**Pull request merged**